src/main.c

```diff
@@ -106,7 +106,6 @@ static int parse_options(int argc, char **argv, int *index)
 			print_help(argv[0]);
 			exit(EXIT_SUCCESS);
 		case 'v':
-			copyright_header("swupd");
```

Comment:
Having the copyright print out when passing `--version` is a common convention, so I think we should keep this one.